### DISCUSSION TITLE
Adding binaries to function runtimes

### DISCUSSION THREAD

**Discussion by JohnValanidas** [vercel/vercel#10142] [Help]
### Problem Description

I have a project that's npm dependency requires GraphicsMagick to be installed so it can transform PDF files into preview images inside of our application. Initially, we were using a package that didn't require any external dependencies, but its compressed runtimes were larger than the 50 Mb allowed. I even attempted to separate this out into its own Vercel function instead of the main GraphQL one, but it was still too large at 59 Mb. 

I built the GraphicsMagick binary during the build phase of the Vercel deployment, but I don't know how to get it packaged into the deployed runtimes. Is this something that I am able to do? It seems there is support for GraphicsMagick on AWS Lambda, but I don't know how well that transfers to Vercel deployments.  

Context: 
- Base Framework: RedwoodJs version 2.0.0
- NPM Package: [pdf2image (pdf2pic on npm)](https://github.com/yakovmeister/pdf2image)
  - [Required Dependency doc](https://github.com/yakovmeister/pdf2image/blob/master/docs/gm-installation.md) has a topic on AWS Lambda functions.

Note: I hope this is the right forum for this question, if it isn't, please let me know and I'll move my question.

**Comment by htmujahid** [comment]
we can actually install in installCommand
```dnf install -y graphicsmagick```
but this does not also solves the problem
